docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -243,6 +243,40 @@ a tarball of the built collection in the current directory which can be uploaded
 This tarball is mainly intended to upload to Galaxy
 as a distribution method, but you can use it directly to install the collection on target systems.
 
+.. _ignoring_files_and_folders_collections:
+
+Ignoring files and folders
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default the build step will include all the files in the collection directory in the final build artifact except for
+
+* ``galaxy.yml``
+* ``*.pyc``
+* ``*.retry``
+* Various version control directories like ``.git/``
+
+To exclude other files and folders when building the collection you can set a list of file glob like patterns in the
+``build_ignore`` key in the collection's ``galaxy.yml`` file. These patterns use the following special characters for
+wildcard matching:
+
+* ``*``: Matches everything
+* ``?``: Matches any single character
+* ``[seq]``: Matches and character in seq
+* ``[!seq]``:Matches any character not in seq
+
+For example if you wanted to exclude the sensitive folder in ``playbooks`` folder as well any ``.tar.gz`` archives you
+can set the following in your ``galaxy.yml`` file:
+
+.. code-block:: yaml
+
+     build_ignore:
+     - playbooks/sensitive
+     - '*.tar.gz'
```

Comment:
This would be so common (builds of past versions), I'd argue that it would be nice to have it in the default ignore list.

Reply:
The trouble with the default ignore list is you cannot overwrite it. This would stop anyone from using a collection that had roles or playbooks that contained these files (and weren't an actual collection release).

Although running `ansible-galaxy collection build` **will** ignore `{namespace}-{name}-*.tar.gz` in the root directory so unless someone is building the collection in another dir it should be ignored on subsequent builds.

Reply:
ok I see how that makes sense now. Looks like the .tar.gz I found was from an alternative namespace, so that's why the rule you mention didn't filter it out.